osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -535,8 +494,10 @@ - (void)showStatus:(enum kaliteStatus)status {
             self.stopButton.enabled = NO;
             self.openBrowserButton.enabled = NO;
             [self.statusItem setImage:[NSImage imageNamed:@"favicon"]];
-            [self.statusItem setToolTip:@"KA-Lite is stopped."];
+            [self.statusItem setToolTip:@"KA Lite is stopped."];
             showNotification(@"Stopped");
+            self.customKaliteData.enabled = YES;
+            [self.customKaliteData setToolTip:@"Select KA Lite data path."];
```

Comment:
Can you add a more descriptive text here?  Or you can remove this if we will implement the help text.